### ISSUE TITLE
Missing focus indicators in banner connect menu [Safari/a11y]

### ISSUE THREAD

**Issue by knowler** [npmx-dev/npmx.dev#611]
The focus indicators for the “connect” menu in the site header are missing in macOS Safari.

https://github.com/user-attachments/assets/ff329839-cc24-423d-b0c0-841d2ec83a30

Tested: Safari 26.2 on macOS Tahoe 26.2

I also noticed that the focus indicators in Firefox and Chrome are kinda janky looking. Could probably use some design love while fixing these.
